hello_world/transactions/test/hello_transaction.test.js

```diff
@@ -0,0 +1,91 @@
+const HelloTransaction = require('../hello_transaction');
```

Comment:
Missing license header

Reply:
which license header?

Reply:
We use this (year need to be 2020)
https://github.com/LiskHQ/lisk-sdk/blob/development/framework/test/jest/unit/specs/application/application.spec.js#L1-L13